test/integration/api/projects.js

```diff
@@ -1180,3 +1180,29 @@ describe('api: /projects', () => {
   });
 });
 
+describe('api: /projects-forms', () => {
+  describe('GET', () => {
+    it('should return projects with nested extended forms', testService((service) =>
+      service.login('alice', (asAlice) => asAlice.get('/v1/projects-forms')
+        // todo: revisit this...
+        // extended=true to get form details but not really needed for projects
```

Comment:
I also think that it'd be nice for performance for the projects query not to hit the `submissions` table. Maybe this endpoint shouldn't require the extended metadata header and should default to extended metadata for forms?